.nais/naiserator-dev.yaml

```diff
@@ -92,6 +92,8 @@ spec:
       value: "false"
     - name: NY_CRONJOB_ENABLED
```

Comment:
Er det noe gamle toggles med tilhørende kode som kan fjernes her?

Reply:
Ja, både `NY_CRONJOB_ENABLED` og `AUTOMATISK_OPPFYLT_CRONJOB_ENABLED` med tilhørende kode kan nok fjernes. Det var noen cronjobber vi kjørte for å rydde i noen feil etter den første prodsettingen.